docs/source/algorithms/IndexSXPeaks-v1.rst

```diff
@@ -25,10 +25,33 @@ transfered between workspaces.
 Usage
 -----
 
-.. warning::
+**Example - a simple example of IndexPeaks**
 
-    This algorithm is deprecated and should no longer be used. One should consider using
```

Comment:
The deprecation warning seems to be gone now. Is it no longer deprecated?

Reply:
@SimonHeybrock It was deprecated by Michael Reuter who is no longer here.  I talked to Owen who wrote algorithm and he said not to deprecate if it still works.